tests/gpflow/models/test_gpr_posterior.py

```diff
@@ -0,0 +1,50 @@
+from itertools import product
```

Comment:
I think it would be useful to test the `update_cache` method on the `AbstractPosterior` base class through this subclass because the cached values have different mathematical meaning for this object.

Reply:
Why is this test file in the `models` package? I would have expected to find these tests in the same place as the SVGP posterior tests.

Reply:
> I think it would be useful to test the update_cache method on the AbstractPosterior base class through this subclass because the cached values have different mathematical meaning for this object.
Have added a test in `test_posteriors.py`

Secondpoint:
It is in the same location, but happy to move them all if you want?

![image](https://user-images.githubusercontent.com/44336612/123668143-bc0f0f00-d832-11eb-981c-b725a65b7373.png)